### PR TITLE
update for tech support role

### PR DESCRIPTION
**Pull request by peterlangmar**
Tension: these issues were commons sense and or already discussed with the tech support role, but it is important to capture them for clarity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,12 +230,21 @@ Domains:
...
+Purpose:
+- Seamless tech work environment
 
 Accountabilities:
 - Fixing tech stuff around office: TV, wifi, printer, updating computers
 - Managing tech infrastructure
-- Removing the accounts of people who leave the company
+- Recharging rechargable batteries
+- Taking used batteries to the appropirate dangerous waste deposit
+- Making the accounts of Partners who leave the company
```
Comment (cassus) [inline README.md]: typo

Comment (peterlangmar) [inline README.md]: @cassus what do you mean exactly?

Comment (cassus) [inline README.md]: "Making the accounts of Partners who _leave_ the company"

-> _join_ ?

Comment (peterlangmar) [inline README.md]: fixed thanks

**Pull request merged**